packages/mobile/src/screens/sign-up-screen/screens/CreateEmailScreen.tsx

```diff
@@ -0,0 +1,214 @@
+import { useCallback, useContext, useMemo } from 'react'
+
+import {
+  AudiusQueryContext,
+  emailSchema,
+  emailSchemaMessages
+} from '@audius/common'
+import { css } from '@emotion/native'
+import { useTheme } from '@emotion/react'
+import { setValueField } from 'common/store/pages/signon/actions'
+import { getEmailField } from 'common/store/pages/signon/selectors'
+import { Formik } from 'formik'
+import { Dimensions, ImageBackground } from 'react-native'
+import RadialGradient from 'react-native-radial-gradient'
+import { useDispatch, useSelector } from 'react-redux'
+import { toFormikValidationSchema } from 'zod-formik-adapter'
+
+import { Box, Flex, Link, Text } from '@audius/harmony-native'
+import DJBackground from 'app/assets/images/DJportrait.jpg'
+import { Button } from 'app/components/core'
+import { TextField } from 'app/components/fields'
+import { useNavigation } from 'app/hooks/useNavigation'
+
+import { AudiusValues } from '../components/AudiusValues'
+import { SocialMediaLoginOptions } from '../components/SocialMediaLoginOptions'
+import { Heading, Page } from '../components/layout'
+import { Divider } from '../components/temp-harmony/Divider'
+import { Hint } from '../components/temp-harmony/Hint'
+import { IconAudiusLogoHorizontalColor } from '../components/temp-harmony/IconAudiusLogoHorizontalColor'
+import IconExclamation from '../components/temp-harmony/IconExclamation.svg'
+import type { SignUpScreenParamList } from '../types'
+
+const messages = {
+  title: 'Sign Up For Audius',
+  emailLabel: 'Email',
+  signUp: 'Sign Up Free',
+  haveAccount: 'Already have an account?',
+  signIn: 'Sign In',
+  subHeader: (
+    <>
+      Join the revolution in music streaming!{'\n'}Discover, connect, and create
+      on Audius.
+    </>
+  ),
+  socialsDividerText: 'Or, get started with one of your socials',
+  unknownError: 'Unknown error occurred.',
+  metaMaskNotRecommended: 'Signing up with MetaMask is not recommended.',
+  signUpMetamask: 'Sign Up With MetaMask',
+  learnMore: 'Learn More'
+}
+
+type SignUpEmailValues = {
+  email: string
+}
+
+export const CreateEmailScreen = () => {
+  const dispatch = useDispatch()
+  const navigation = useNavigation<SignUpScreenParamList>()
+  const existingEmailValue = useSelector(getEmailField)
+  const queryContext = useContext(AudiusQueryContext)
```

Comment:
useAudiusQueryContext will ensure context is always there, so we dont have to do `if (queryContext)` below